Spanner/src/SpannerClient.php

```diff
@@ -566,6 +578,14 @@ function (array $instance) {
      * $database = $spanner->connect('instance-id', 'database-id');
      * ```
      *
+     * ```
+     * Database role configured on the $options parameter
+     * will be applied to the session created by this object.
+     * Note: When the databseRole and sessionPool both are present in the options,
+     * we prioritize the sessionPool.
+     * $database = $spanner->connect('instance-id', 'database-id', ['databaseRole' => 'Reader']);
+     * ```
```

Comment:
These comments will be treated as a code by the snippet extractor. You may want to move the code block start (triple tilde) from line 581 to 585.
```suggestion
     * Database role configured on the $options parameter
     * will be applied to the session created by this object.
     * Note: When the databseRole and sessionPool both are present in the options,
     * we prioritize the sessionPool.
     * ```
     * $database = $spanner->connect('instance-id', 'database-id', ['databaseRole' => 'Reader']);
     * ```
```

Reply:
Done